samples/attested_tls/server/enc/cert_verifier.cpp

```diff
@@ -59,13 +60,14 @@ int cert_verify_callback(
     if (cert_size <= 0)
         goto exit;
 
-    result = oe_verify_attestation_certificate(
-        cert_buf, cert_size, enclave_identity_verifier_callback, NULL);
+    oe_verifier_initialize();
```

Comment:
Can we move oe_verifier_initialize() to somewhere else so it only gets called once?

Reply:
Sure. Thanks for point that out.
Updated.